Tests/Unit/EmogrifierTest.php

```diff
@@ -750,6 +756,43 @@ public function emogrifyFormatsCssDeclarations($cssDeclarationBlock, $expectedSt
     }
 
     /**
+     * Data provider for emogrifyInvalidDeclaration.
+     *
+     * @return array[]
+     */
+    public function invalidDeclarationDataProvider()
+    {
+        return [
+            'one declaration with missing dash in property name' => ['font weight: bold;', 'weight:'],
+            'one declaration with invalid character in property name' => ['-9webkit-text-size-adjust:none;', 'webkit'],
+            'one declaration with missing :' => ['-webkit-text-size-adjust none', 'webkit'],
+            'one declaration with missing value' => ['-webkit-text-size-adjust :', 'webkit'],
+        ];
+    }
+
+    /**
+     * @test
+     *
+     * @param string $cssDeclarationBlock the CSS declaration block (without the curly braces)
+     * @param string $notExpectedStyleAttributeContent the value that should not be withint the style attribute
+     *
+     * @dataProvider invalidDeclarationDataProvider
+     */
+    public function emogrifyInvalidDeclaration($cssDeclarationBlock, $notExpectedStyleAttributeContent)
```

Comment:
Please let's make the test names sentences about the behavior. My suggestion: emogrifyDropsInvalidCssDeclarations